onnxruntime/core/framework/session_state.h

```diff
@@ -38,6 +38,7 @@
 #include "core/framework/memory_info.h"
 #endif
 
+#include "core/framework/cloud_invoker.h"
```

Comment:
Why does session state need to know this?

Reply:
Removed.